components/builder-api/habitat/default.toml

```diff
@@ -1,3 +1,6 @@
+[srv]
+port = 9636
```

Comment:
Oh I see, you just are suggesting to use a different namespace in the toml for our config than `srv`. Can do that for sure. It's not coming to my mind immediately though, where else do we use `srv` in config namespaces?